apps/condo/domains/acquiring/schema/RegisterMultiPaymentForVirtualReceiptService.js

```diff
@@ -140,7 +140,7 @@ const RegisterMultiPaymentForVirtualReceiptService = new GQLCustomSchema('Regist
 
                 const acquiringIntegration = await AcquiringIntegration.getOne(context, {
                     id: acquiringContext.integration,
-                })
+                }, 'id minimumPaymentAmount maximumPaymentAmount hostUrl deletedAt')
```

Comment:
I guess its better to move this to some const on the top of the file, like we do with `ERRORS`.

Reply:
It's better to keep it close to use. It's easier to review. Moving it to another file will decrease readability because you need to look at another file.

Reply:
I prefer to keep fields along with getOne/getAll and other server methods at the same line of code.
I believe we have to not move it to top of the same file and not move it to another file - since it more readable to see exactly fields you are selecting from a model. More over - we going to never reuse those constants in future to not mix up different queries together

Reply:
> It's better to keep it close to use. It's easier to review. Moving it to another file will decrease readability because you need to look at another file.

I meant not another file, but the top of existing one. Like we do with ERRORS. So you can avoid horizontal scroll

Reply:
I would leave the inline style, it makes the code easier to review.
You can extract some long query into a constant, but only to improve readability